packages/Popover/src/Popover.tsx

```diff
@@ -139,15 +139,15 @@ interface PopoverProps {
  * @param props.adjustOnMutation Should the Popover auto adjust its content when the DOM changes (using MutationObserver).
  */
 function Popover({
-  children,
-  active,
+  children = undefined,
```

Comment:
This is undefined by default (due to JS), seems un-necessary to include.

Reply:
Kk removing!